req.py

```diff
@@ -1,334 +1,351 @@
+            case 'addy':
+                info = fake.street_address()
+            case 'city':
+                info = city
+            case 'zip':
+                zipp = zip_codes[city]
+                info = zipp[zip_num]
+            case 'job':
+                info = fake.job()
+            case 'salary':
+                info = random.randint(15, 35)
+
+        driver.find_element_by_xpath(data2.get(key)).send_keys(info)
+
+    print(f"successfully filled out app forms for {city}")
+
+    # fill out dropdowns
+    select = Select(driver.find_element_by_id('154:_select'))
+    select.select_by_visible_text('Yes')
+    select = Select(driver.find_element_by_id('195:_select'))
+    select.select_by_visible_text('United States')
+
+    select = Select(driver.find_element_by_id('211:_select'))
+    select.select_by_visible_text('Yes')
+    select = Select(driver.find_element_by_id('215:_select'))
+    select.select_by_visible_text('No')
+    select = Select(driver.find_element_by_id('219:_select'))
+    select.select_by_visible_text('No')
+    select = Select(driver.find_element_by_id('223:_select'))
+    select.select_by_visible_text('No')
+    select = Select(driver.find_element_by_id('227:_select'))
+    select.select_by_visible_text('No')
+    select = Select(driver.find_element_by_id('231:_select'))
+    select.select_by_visible_text('Yes')
+    select = Select(driver.find_element_by_id('223:_select'))
+    select.select_by_visible_text('No')
+
+    time.sleep(1)
+
+    select = Select(driver.find_element_by_id('235:_select'))
+    gender = random.choice(['Male', 'Female', 'Other'])
+    select.select_by_visible_text(gender)
+
+    driver.find_element_by_xpath('//label[text()="350 LBS"]').click()
+    driver.find_element_by_xpath('//label[text()="800 LBS"]').click()
+    els = driver.find_elements_by_xpath('//label[text()="Yes"]')
+    for el in els:
+        el.click()
+
+    time.sleep(5)
+    driver.find_element_by_xpath('//*[@id="261:_submitBtn"]').click()
+    print(f"successfully submitted application")
+
+
+def main():
+    i = 1
+    while (i < 10000):
+        rand_num = random.randint(0, 3)
+        try:
+            driver = start_driver(rand_num)
+        except Exception as e:
+            print(f"failed to start driver: {str(e)}")
+            driver.close()
+            continue
+
+        time.sleep(2)
+
+        try:
+            generate_account(driver, rand_num)
+        except Exception as e:
+            print(f"failed to create account: {str(e)}")
+            driver.close()
+            continue
+
+        try:
+            fill_out_application_and_submit(driver, rand_num)
+        except Exception as e:
+            print(f"failed to fill out app and submit: {str(e)}")
+            driver.close()
+            continue
+
+        driver.close()
+        time.sleep(5)
+
+
+if __name__ == '__main__':
+    t1 = threading.Thread(target=main)
+    t2 = threading.Thread(target=main)
```

Comment:
Great implementation. One thing to note: python's threading module will not allow you to run both of threads simulataneously. Sadly, only multiprocessing and futures support this. You can read about this here. https://docs.python.org/3/library/threading.html